src/test/java/com/coveo/pushapiclient/UpdateStreamServiceInternalTest.java

```diff
@@ -63,15 +78,19 @@ public void addOrUpdateShouldCreateFileContainer() throws IOException, Interrupt
   }
 
   @Test
-  public void addOrUpdateAndDeleteShouldAddDocumentsToQueue()
+  public void addOrUpdateAndPartialAndDeleteShouldAddDocumentsToQueue()
```

Comment:
Now that there are three actions, would it be cleaner to split the test per action?

Reply:
Because we have the possibility to add all 3 types of documents to a single queue I wanted to keep them all in the same test method